src/test/scala/org/graphframes/lib/ParallelPersonalizedPageRankSuite.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.graphframes.lib
+
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.types.DataTypes
+import org.apache.spark.sql.Row
+import org.apache.spark.ml.linalg.{SQLDataTypes, SparseVector}
+
+import org.graphframes.examples.Graphs
+import org.graphframes.{GraphFrameTestSparkContext, SparkFunSuite, TestUtils}
+
+class ParallelPersonalizedPageRankSuite extends SparkFunSuite with GraphFrameTestSparkContext {
+
+  val n = 100
+
+  private def isLaterVersion(minVer: String): Boolean = {
+    val (minVersionMajor, minVersionMinor) = majorMinorVersion(minVer)
+    if (sparkVersionMajor != minVersionMajor) {
+      return sparkVersionMajor > minVersionMajor
+    } else {
+      return sparkVersionMinor >= minVersionMinor
+    }
+  }
+
+  test("Illegal function call argument setting") {
+    val g = Graphs.star(n)
+    val vertexIds: Array[Any] = Array(1L, 2L, 3L)
+
+    def checkPageRankRun(prc: ParallelPersonalizedPageRank): Unit = {
+      if (isLaterVersion("2.1")) {
+        intercept[IllegalArgumentException] { prc.run() }
+      } else {
+        intercept[NotImplementedError] { prc.run() }
+      }
+    }
+
+    lazy val prcSansNumIter = g.parallelPersonalizedPageRank
```

Comment:
Don't use "sans;" use plain English to keep the code as legible by as many people as possible